libs/ledger/include/ledger/chain/block_coordinator.hpp

```diff
@@ -60,66 +68,6 @@ class BlockSinkInterface;
- *           ▼                         ▼                        │                │
- * ┌──────────────────┐      ┌──────────────────┐               │                │
- * │  Schedule Block  │      │Execute New Block │               │                │
- * │    Execution     │      │                  │               │                │
- * └──────────────────┘      └──────────────────┘               │                │
- *           │                         │                        │                │
- *           │                         │                        │                │
- *           ▼                         ▼                        │                │
- * ┌──────────────────┐      ┌──────────────────┐               │                │
- * │Wait for New Block│      │Wait for Execution│               │                │
- * │    Execution     │◀ ─   │                  │◀ ─            │                │
- * └──────────────────┘   │  └──────────────────┘   │           │                │
- *           │                         │                        │                │
- *           │─ ─ ─ ─ ─ ─ ┘            │─ ─ ─ ─ ─ ─ ┘           │                │
- *           ▼                         ▼                        │                │
- * ┌──────────────────┐      ┌──────────────────┐               │                │
- * │ Post Exec. Block │      │   Proof Search   │               │                │
- * │    Validation    │      │                  │◀ ─            │                │
- * └──────────────────┘      └──────────────────┘   │           │                │
- *           │                         │                        │                │
- *           │                         │─ ─ ─ ─ ─ ─ ┘           │                │
- *           │                         ▼                        │                │
- *           │               ┌──────────────────┐               │                │
- *           │               │  Transmit Block  │               │                │
- *           │               │                  │               │                │
- *           │               └──────────────────┘               │                │
- *           │                         │                        │                │
- *           └─────────────────────────┼────────────────────────┘                │
- *                                     │                                         │
- *                                     │                                         │
- *                                     │                                         │
- *                                     ▼                                         │
- *                           ┌──────────────────┐                                │
- *                           │      Reset       │                                │
- *                           │                  │────────────────────────────────┘
- *                           └──────────────────┘
```

Comment:
Why delete this diagram? It is quite useful to understand the code.

Reply:
It needs to be rewritten with extra states

Reply:
Done